garden/install_windows.md

```diff
@@ -5,7 +5,7 @@ WARNING: Current Windows support is experimental.
 # Binary Installation on Windows 10
 
 Most Gazebo packages are available in Windows 10 using the [conda-forge package manager](https://conda-forge.org/),
-and the Gazebo feedstock recipes can be found [here](https://github.com/conda-forge?q=libignition&type=&language=).
+and the Gazebo feedstock recipes can be found [here](https://github.com/search?q=org:conda-forge+libgz&type=code).
```

Comment:
I think this update looks right, but I'm not familiar with Conda packages. Maybe @j-rivero?

Reply:
So if you go to that old link, it does not work, go to the new link and it actually shows you. Github changed the way things can be queried.